lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_node_lowering.cpp

```diff
@@ -348,6 +358,19 @@ class TSNodeLowering : public NodeLowering {
         {input.shape(), weight.shape(), weight.shape()});
   }
 
+  static lazy_tensors::Shape InferNllLossBackward(const ir::Node* node) {
```

Comment:
This part actually doesn't work. I'm not quite sure how to do shape inferences. And therefore upload a WIP to discuss.

Reply:
Have a look at: https://github.com/pytorch/pytorch/blob/5f997a7d2fcd81584d1c9f6e173e30c867892ee8/aten/src/ATen/native/LossNLL.cpp#L14

Reply:
Thanks @asuhan for the pointer and @desertfire for the detailed elaboration.